packages/components/src/core/styles/common/mixins/a11y.ts

```diff
@@ -1,8 +1,17 @@
 export function focusVisibleA11yStyle() {
   return `
       &.Mui-focusVisible, &:focus-visible {
-        outline: 5px auto Highlight;
-        outline: 5px auto -webkit-focus-ring-color;
+        outline: 2px solid black;
+        outline-offset: 1px;
       }
   `;
 }
+
+export function focusA11yStyle() {
```

Comment:
seems like this function is no longer used 😄 ?

Reply:
Oops, I'll remove it :D